source/blog/2015-10-29-announcing-capablanca-release-candidate-1.html.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Announcing Capablanca Release Candidate 1
+date: 2015-10-29 14:03 UTC
+author: chessbyte
+tags: release Capablanca
+comments: true
+published: true
+---
+We're pleased to announce that, effective today, the first [Capablanca release candidate](http://manageiq.org/download/devel/) is now available. Among the improvements in this release:
```

Comment:
@bproffitt 

Unless I'm misunderstanding or there's more website changes coming... but I think this is wrong.  This links to a page with links to to each of the image platforms, which for vmware, links to http://manageiq.org/download/manageiq-vsphere-devel.ova which downloads the latest nightly build off of master:

```
wget http://manageiq.org/download/manageiq-vsphere-devel.ova
...
Location: http://releases.manageiq.org/manageiq-vsphere-master-20151026-d5305dacb7.ova
...
Location: http://82396214b15770b3002a-d3fd4008af5431479c47090ec0cc5a4d.r13.cf5.rackcdn.com//manageiq-vsphere-master-20151026-d5305dacb7.ova
```

Perhaps we need to link to the manageiq-openstack-capablanca-1-rc1 images here:  http://releases.manageiq.org/?